app/assets/stylesheets/navbar.scss

```diff
@@ -28,4 +28,22 @@
   .navbar-collapse {
     border-top: 0;
   }
+
+  @media screen and (min-width: 768px) and (max-width: 991px){
+    .navbar-nav > li {
+      font-size: 12px;
+    }
+
+    .navbar-brand {
+      $brand-height-small: 25px;
+      $vpad: ($navbar-height - $brand-height-small) / 2;
+
+      padding: floor($vpad) 10px ceil($vpad);
+
+      & > img {
```

Comment:
`& > ` are not necessary here since there is only one `img` inside `.navbar-brand`.